tapdb/asset_minting.go

```diff
@@ -1499,6 +1568,8 @@ func (a *AssetMintingStore) AddSproutsToBatch(ctx context.Context,
 
 	var writeTxOpts AssetStoreTxOptions
 	return a.db.ExecTx(ctx, &writeTxOpts, func(q PendingAssetStore) error {
+		// Upsert the assets with genesis and get the database ID of the
```

Comment:
We could add the `upsertGenesisPoint` call to `insertMintAnchorTx` function as well. That would mean it would be called twice in this specific case (since `upsertAssetsWithGenesis` also does that), but we'd be able to remove two instances of it further above...